test/Feature/MakeSymbolicAPI.c

```diff
@@ -5,15 +5,24 @@
 // RUN: FileCheck %s -check-prefix=CHECK-ERR --input-file=%t.stderr.log
 
 int main() {
-  unsigned a, b, c;
+  unsigned a, b, c, d, e;
+  const char *invalid_pointer = 0xf;
 
   klee_make_symbolic(&a, sizeof(a), "");
 // CHECK-WRN: KLEE: WARNING: klee_make_symbolic: renamed empty name to "unnamed"
 
+
   klee_make_symbolic(&b, sizeof(b));
 // CHECK-WRN: KLEE: WARNING: klee_make_symbolic: deprecated number of arguments (2 instead of 3)
 // CHECK-WRN: KLEE: WARNING: klee_make_symbolic: renamed empty name to "unnamed"
 
+  if(a == 2)
+    klee_make_symbolic(&d, sizeof(e), invalid_pointer);
+// CHECK-ERR-DAG: KLEE: ERROR: {{.*}} Invalid string pointer passed to one of klee_ functions
```

Comment:
I would align this with the `klee_make_symbolic` call that generates the message (and the same in the rest of the file)